feedparser/encodings.py

```diff
@@ -208,7 +208,7 @@ def convert_to_utf8(http_headers, data, result):
     application_content_types = ('application/xml', 'application/xml-dtd',
                                  'application/xml-external-parsed-entity')
     text_content_types = ('text/xml', 'text/xml-external-parsed-entity')
-    json_content_types = ('application/feed+json', 'application/json')
+    json_content_types = ('application/json', 'application/feed+json')
```

Comment:
changing the order in this tuple is actually enough to fix everything since on line 238 we do:
```python
    http_content_type = json_content_types[0]
```
which makes code harder to understand imho yet I wanted to propose the smallest set of changes possible in order to get this merged quickly.